ports/atmel-samd/boards/circuitbrains_basic_m0/mpconfigboard.mk

```diff
@@ -0,0 +1,17 @@
+LD_FILE = boards/samd21x18-bootloader-external-flash-crystalless.ld
```

Comment:
Obsolete; remove `LD_FILE` definition.

Reply:
Cleaned up board definitions per this conversation.